core/src/main/java/io/questdb/network/AbstractIODispatcher.java

```diff
@@ -58,7 +59,8 @@
     protected final RingQueue<IOEvent<C>> ioEventQueue;
     protected final MCSequence ioEventSubSeq;
     protected final NetworkFacade nf;
-    protected final ObjLongMatrix<C> pending = new ObjLongMatrix<>(4);
+    protected final ObjLongMatrix<C> pending = new ObjLongMatrix<>(5);
```

Comment:
can we pull this out into configuration pls?

Reply:
it is a non-configurable parameter, refactored to a named constant

Reply:
👍